pkgs/by-name/va/vapoursynth-wnnm/package.nix

```diff
@@ -0,0 +1,72 @@
+{
+  lib,
+  gcc12Stdenv,
+  fetchFromGitHub,
+  cmake,
+  pkg-config,
+  mkl,
+  vapoursynth,
+  zimg,
+}:
+
+gcc12Stdenv.mkDerivation (finalAttrs: {
```

Comment:
Why the restriction to GCC 12? I understand that GCC 13 may produce incorrect code with CUDA, but this specific package does not rely on it.